linux/setup_repo.sh

```diff
@@ -7,6 +7,32 @@ if [[ ${EUID} -ne 0 ]]; then
      exit 1
 fi
 
+# Check for apt version >= 2.4.0 (if found, assuming Debian-based compatible with repo)
+# Issue: https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1950095
+
+if APT_VERSION="$(apt --version)"; then
+    APT_MAJOR=$(echo $APT_VERSION| cut -d' ' -f 2 | cut -d'.' -f 1)
+    APT_MINOR=$(echo $APT_VERSION| cut -d' ' -f 2 | cut -d'.' -f 2)
+else
+    echo "This script is only compatible with Debian based distributions which have apt, but apt is not available. Please check that your OS is supported."
+    exit 1
+fi;
+
+if (( $(echo "${APT_MAJOR}.${APT_MINOR} < 2.2" | bc -l) )); then
+    echo "Your apt version is incompatible. You may not be able install this repository. "
+    echo "See issue: https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1950095"
+    echo "Please update your OS or use the .deb package from the Website to manually install Jamulus."
+    echo "Do you wish to install anyway?"
```

Comment:
```suggestion
    echo "Do you wish to install anyway (Yes/No)?"
```

Reply:
The yes/no already appears on screen so there isn't any need to add this.